packages/react-dev-utils/formatBuildError.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const get = require('lodash/get');
+const chalk = require('chalk');
+
+module.exports = function formatBuildError(err) {
+  const message = get(err, 'message');
+
+  // Add more helpful message for UglifyJs error
+  if (typeof message === 'string' && message.indexOf('from UglifyJs') !== -1) {
+    try {
+      console.log(
+        'Failed to minify the code from \n\n',
+        chalk.yellow(
+          /Unexpected token:(.+)\[(.+)\]\[(.+)\]/.exec(err.stack)[2]
```

Comment:
like this?  ![screen shot 2017-06-29 at 10 24 52 pm](https://user-images.githubusercontent.com/9636410/27696118-051e0e2c-5d1b-11e7-8171-a5899b0d2ee5.png)